cnf-tests/hack/compare-gen-md.sh

```diff
@@ -4,8 +4,9 @@
 
 MDFILE="TESTLIST.md"
 
-mv cnf-tests/${MDFILE} _cache/${MDFILE}.old
+mv ${MDFILE} _cache/${MDFILE}.old
 
-. $(dirname "$0")/generate-cnf-docs.sh
+ls
```

Comment:
is this doing anything?

Reply:
nope, that was for debugging purpouses, thanks for spotting it!